IdentityCore/src/webview/systemWebview/session/MSIDSystemWebviewController.m

```diff
@@ -132,6 +132,10 @@ - (void)startWithCompletionHandler:(MSIDWebUICompletionHandler)completionHandler
 #endif
     void (^authCompletion)(NSURL *, NSError *) = ^void(NSURL *callbackURL, NSError *authError)
     {
+#if TARGET_OS_IPHONE
```

Comment:
I think we should call stop operation in [notifyEndWebAuthWithURL:] as it is called here and in [self cancel] etc. so it will cover scenario when user cancels the webview as well.

Reply:
+1

Reply:
I was finding in my testing that the completion handler was being called every time when I was cancelling the webview from the "cancel" button.

That being said, I think it's safer to just move it and be sure, so I've moved it as you suggested. Thanks!